site/src/pages/community/developer-guide.mdx

```diff
@@ -386,3 +386,52 @@ try {
     assertTrue(e.getMessage().contains("bad method"));
 }
 ```
+
+## How to write pull request description
+
+There are three sections you need to write: `Motivation`, `Modifications`, and `Result`.
+
+### Motivation
+
+Explain why you're sending the pull request (PR) and what's the problem you're trying to solve.
+You do not have to include all the detail but please include as much as the reviewers can get the background
+of this change.
+If there's related GitHub issues, please link to them.
```

Comment:
Should be
-> If there are related GitHub issues ?

Reply:
Thanks, fixed. 😉